manifests/site.pp

```diff
@@ -8,6 +8,10 @@
 
 ### global defaults ###
 
+# We almost always intend to create system users/groups
+User { system => true, groups => ['sys'] }
```

Comment:
Members of sys group are allowed access in [access.conf](https://github.com/ocf/puppet/blob/master/modules/ocf/templates/access.conf.erb), permitting cron jobs and logins. This OCFism dates back to 2012. Debian doesn't use the sys group for anything.

Reply:
Since every non-sorried user is now allowed cron access, you only need the sys group to allow logins as that user (including through sudo). Perhaps consider getting rid of the sys group and, if necessary, allowing every non-sorried user to log in from LOCAL in addition to cron/crond.